build/ProjectReunion-BuildFoundation-Weekly.yml

```diff
@@ -31,6 +31,9 @@ parameters:
   - name: "enableLicenseInstall"
     type: boolean
     default: true
+  - name: "pipelineType"

```

Comment:
hmm this is a parameter but $(PipelineType) it is intending to fill is a variable?

Reply:
I'm curious why it appears to have fixed the error.
Sure, I can try a variable.